### DISCUSSION TITLE
How to report a scamming user

### DISCUSSION THREAD

**Discussion by jbanks070707** [community/community#108120] [New to GitHub]
Hi all,
How do I report a scamming user to Github ?

**Comment by mw-xyz** [comment]
[This link](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#) should help.

**Comment by queenofcorgis** [comment]
Hi @jbanks070707 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
